AutomatticTracks/src/main/java/com/automattic/android/tracks/DeviceInformation.java

```diff
@@ -242,14 +242,18 @@ public Boolean isWifiConnected() {
     }
 
     public Boolean isBluetoothEnabled() {
+        // Do not retrieve bluetooth info on older Android versions.
+        // See: https://github.com/Automattic/Automattic-Tracks-Android/issues/20
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
+            return null;
+        }
+
         Boolean isBluetoothEnabled = null;
-        try {
+        if (PackageManager.PERMISSION_GRANTED == mContext.checkCallingOrSelfPermission(Manifest.permission.BLUETOOTH)) {
```

Comment:
From the doc:
> checkCallingOrSelfPermission: Determine whether the calling process of an IPC or you have been granted a particular permission. This is the same as checkCallingPermission(String), except it grants your own permissions if you are not currently processing an IPC. Use with care!

I think it's safer to keep the old code (catch `SecurityException `)